### ISSUE TITLE
Annotation problem

### ISSUE THREAD

**Issue by JieFeng-cse** [epic-kitchens/action-models#5]
Hi,
It's very nice of you to provide the weights for us.
I have a naive question that the annotation you use while training is the class-key or the exact verb?
I want to use it as plug-in so I may not so familiar with the background knowledge, can you help me?

**Comment by willprice** [comment]
Hi @aplumpcslearner,
The class key of the verb (i.e. the canonical name of the verb cluster). There are 124 outputs from the verb layer and 352 from the noun layer.

Does that answer your question?

**Comment by JieFeng-cse** [comment]
yes, perfectly answered.

**Comment by nirat1606** [comment]
Hi, 

I still have a question about this. Is the annotation file same as https://github.com/epic-kitchens/annotations/blob/master/EPIC_verb_classes.csv or the indices of verb classes are alphabetically ordered for logits?

Thanks,
Nirat

**Comment by willprice** [comment]
The logits are in the same order as the in the annotation file.

**Comment by nirat1606** [comment]
Thanks for the response. Can you also share somewhere the list of test videos seen and unseen which are used for reporting the numbers on these baselines?
